src/main/java/com/jcraft/jsch/ChannelSftp.java

```diff
@@ -32,6 +32,9 @@
 import java.util.Hashtable;
 import java.util.Vector;
 
+import com.jcraft.jsch.Channel.MyPipedInputStream;
```

Comment:
This is unnecessary.
Since `ChannelSftp` extends `Channel`, it can directly reference the `MyPipedInputStream` class without an import statement.

Reply:
Removed the import as it was not needed. Thanks for the suggestions.